### PR TITLE
LPS-74468 "Get URL" and "WebDAV URL" fields are not selected on iOS on tap

### PR DESCRIPTION
**Pull request by IstvanD**
Hey @p2kmgcl 

@ambrinchaudhary asked me in https://github.com/ambrinchaudhary/liferay-portal/pull/170#issuecomment-327450086 to send this pull request to you for reviewing.

This issue is reproducible only on an iPhone or a iPhone emulator like the one in the XCode development environment.

I implemented a solution that is based on the workaround presented in https://stackoverflow.com/questions/3272089/programmatically-selecting-text-in-an-input-field-on-ios-devices-mobile-safari

In access_from_desktop.jsp, I had to change

var webdavURLInput = webdavDialog.get('boundingBox').one('.webdav-url-resource');

to

var webdavURLInput = webdavDialog.get('boundingBox').one('#<portlet:namespace />webDavURL');

because the aui:input tag doesn't retains it's cssClass="webdav-url-resource" attribute after rendering.
webdavURLInput.focus(); would throw an exception.

The aui:input tag will be rendered to this form

<input class="form-control form-text lfr-input-resource " id="_com_liferay_document_library_web_portlet_DLAdminPortlet_webDavURL" onclick="this.select();" readonly="true" type="text" value="http://localhost:8080/webdav/guest/document_library/folder" tabindex="-1">

therefore, I used the tag element's ID to select it.

Thank you,
István

### CODE REVIEW

File: modules/apps/collaboration/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/access_from_desktop.jsp [hunk 1]
```diff
@@ -92,9 +92,18 @@ else {
...
+						var webdavURLInput = webdavDialog.get('boundingBox').one('#<portlet:namespace />webDavURL');
 
 						webdavURLInput.focus();
+
+						webdavURLInput.on(
+							'click',
+							function(event) {
+								var webdavURLInputDOMNode = this.getDOMNode();
+
+								webdavURLInputDOMNode.setSelectionRange(0, 9999);
```
Comment (p2kmgcl) [inline modules/apps/collaboration/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/access_from_desktop.jsp]: @IstvanD can you tell me how can I test this change? I have tested the other templates, but can't find where is this one acting.

Comment (IstvanD) [reply modules/apps/collaboration/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/access_from_desktop.jsp]: Hey p2kmgcl

Please create a folder in Document Library, and then select "Access from desktop" from it's drop down menu.
Then a pop-up should appear with the WebDAV URL.

Hope it helps.
István

Comment (p2kmgcl) [reply modules/apps/collaboration/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/access_from_desktop.jsp]: :+1: perfect! Thanks so much

Comment (IstvanD) [reply modules/apps/collaboration/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/access_from_desktop.jsp]: Sure.
Please let me know if you have further questions.

**Comment by p2kmgcl** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by p2kmgcl** [comment]
https://github.com/ambrinchaudhary/liferay-portal/pull/172

**Pull request closed without merging**